bee-inx/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "bee-inx"
+version = "0.1.0"
+authors = [ "IOTA Stiftung" ]
+edition = "2021"
+description = "Rust bindings for IOTA node extensions (INX)"
+readme = "README.md"
+repository = "https://github.com/iotaledger/bee"
+license = "Apache-2.0"
+keywords = [ "iota", "plugin", "node", "inx", "bee" ]
+homepage = "https://www.iota.org"
+
+[dependencies]
+bee-block = { path = "../bee-block", version = "1.0.0-beta.3", default-features = false, features = [ "inx", "std" ] }
```

Comment:
Why are we giving it the full beta release version again? This implies a lot of unnecessary changes when the dependency bumps the index, but I don't see the advantage of doing it. Only locking it to `version = "1"` would actually change behavior of the compiler.

Reply:
The other Bee crates do it the same way. If we were to change it here, we should change it everywhere. I've flipped `version` and `path` to be more consistent though.

Reply:
I am closing this, but I'll open an issue to get clear on that. It doesn't seem to be THAT obvious :slightly_smiling_face: